app-backup/buttersink/buttersink-0.6.8.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="rsync-like utility for btrfs snapshots"
+HOMEPAGE="https://github.com/AmesCornish/buttersink"
+
+LICENSE="GPL-3"
+SLOT=0
+
+# tests require network access
+RESTRICT="test"
+
+if [[ "${PV}" == 9999 ]] ; then
+	inherit git-r3
+	KEYWORDS=""
+	EGIT_REPO_URI="https://github.com/AmesCornish/buttersink"
+	SRC_URI=""
+else
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/AmesCornish/buttersink/archive/${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/boto
+	dev-python/crcmod
+	dev-python/flake8
+	dev-python/psutil
+	sys-fs/btrfs-progs"
+DEPEND="${RDEPEND}"
+
+python_prepare()
+{
+	if [[ ${PV} == 9999 ]] ; then
+		emake makestamps
+		emake buttersink/version.py
+	else
+		mkdir makestamps
```

Comment:
If either of those commands don't work, it'll `die` anyway immediately afterwards due to the lack of git files in the tarball. Added the `|| die` though. I used `mkdir` because that's what the make target does, but `touch` works too so long as the tests aren't being run.